src/main/java/com/metamx/metrics/SysMonitor.java

```diff
@@ -268,10 +275,12 @@ public void emit(ServiceEmitter emitter)
               log.error(e, "Failed to get FileSystemUsage[%s]", name);
             }
             if (fsu != null) {
-              final Map<String, Long> stats = ImmutableMap.of(
-                  "sys/fs/max", fsu.getTotal() * 1024,
-                  "sys/fs/used", fsu.getUsed() * 1024
-              );
+              final Map<String, Long> stats = ImmutableMap.<String, Long>builder()
```

Comment:
I think stats like diskQueue/diskServiceTime are also worth to add (you can find them in FileSystemUsage or DiskUsage, probably adding them under disk umbrella makes more sense).